inc/plugin.class.php

```diff
@@ -1748,7 +1769,7 @@ public static function setLoaded($id, $name) {
     */
    public static function setUnloaded($id) {
       global $GLPI_CACHE;
-      $plugins = $GLPI_CACHE->get('plugins');
+      $plugins = $GLPI_CACHE->get('plugins', []);
```

Comment:
This will prevent any problem if call is made prior to cache key initialization (this should not happen).